src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/InstanceRestoreOnly.kt

```diff
@@ -0,0 +1,126 @@
+package com.cognifide.gradle.aem.instance.tasks
+
+import com.cognifide.gradle.aem.common.AemException
+import com.cognifide.gradle.aem.common.AemTask
+import com.cognifide.gradle.aem.common.Formats
+import com.cognifide.gradle.aem.common.file.FileOperations
+import com.cognifide.gradle.aem.common.file.resolver.FileResolver
+import com.cognifide.gradle.aem.common.onEachApply
+import com.cognifide.gradle.aem.common.tasks.LocalInstanceTask
+import com.cognifide.gradle.aem.instance.InstanceException
+import com.cognifide.gradle.aem.instance.LocalInstance
+import com.cognifide.gradle.aem.instance.names
+import com.fasterxml.jackson.annotation.JsonIgnore
+import java.io.File
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.TaskAction
+
+open class InstanceRestoreOnly : LocalInstanceTask() {
```

Comment:
why do we have `RestoreAndUp` and `RestoryOnly` ? confusing...

Reply:
to be removed